crates/block-producer/src/lib.rs

```diff
@@ -51,10 +51,10 @@ const SERVER_MEMPOOL_EXPIRATION_SLACK: u32 = 2;
 const CACHED_MEMPOOL_STATS_UPDATE_INTERVAL: Duration = Duration::from_secs(5);
 
 /// How often a block is created.
-pub const DEFAULT_BLOCK_INTERVAL: Duration = Duration::from_secs(5);
+pub const DEFAULT_BLOCK_INTERVAL: Duration = Duration::from_secs(2);
```

Comment:
After thinking some more about this, let's reduce this to 3 seconds. We will still likely to go to 2 seconds, but I'd like a bit more time to think it through.